test/schematic_test.exs

```diff
@@ -105,6 +177,48 @@ defmodule SchematicTest do
       assert {:ok, %{"foo" => "hi there!"}} == unify(schematic, input)
     end
 
+    test "map/1 schematic options properties" do
```

Comment:
yeah! i had trouble with it though because using it imports everything from `StreamData` into the body which gave me naming conflicts with `Schematic` functions. i guess we could not `import Schematic` at the top of the test module, or we could just use fully qualified names in the few places where there are conflicts. 

if you think either of those would read better then i’m down to make the changes

Reply:
are you sure? I think that is what `use ExUnitProperties` does, and you are already calling that.

I think that the property macro is just the same as the test macro, but registers them as properties so you can do something like `mix test --only properties`

Reply:
yeah maybe i did something wrong, and then just moved on without double checking. i’ll try again 🙂 thanks for pointing it out, as i forgot to mention it in my description for the PR

Reply:
you were right on all counts! (except that the mix command is `mix test --only property`, in the singular). committed the change [here](https://github.com/clark-lindsay/schematic/commit/316f3c462875b1bc0330205778a672576a4b8738)

thanks again for pointing this out so that i could double back and make this improvement. not sure what mistake i made earlier that led me to believe that that was going to be more trouble than it was worth.